vllm/model_executor/models/siglip.py

```diff
@@ -510,7 +509,8 @@ def __init__(
             config,
             quant_config,
             num_hidden_layers_override=num_hidden_layers_override,
-            need_post_layernorm=self.need_post_layernorm)
+        )
+        self.need_post_layernorm = self.vision_model.need_post_layernorm
```

Comment:
We make this a property.

Reply:
Good idea. I make `need_post_layernorm` for `SiglipVisionModel` as a property.